docs/DeveloperGuide.md

```diff
@@ -263,35 +263,34 @@ _{Explain here how the data archiving feature will be implemented}_
 | `* * *`  | SoC professor                          | view the list of all students taking their FYP under me                                                                                 | get to know more information about my own students                           |
-| `* *`    | responsible student                    | check my upcoming deadlines with an option to see yet-to-submit deliverables                                                            | avoid missing any submission deadlines by accident [coming soon]             |
+| `* *`    | responsible FYP student                | check my upcoming deadlines with an option to see yet-to-submit deliverables                                                            | avoid missing any submission deadlines by accident [coming soon]             |
 | `* *`    | SoC professor                          | view the progress updates that my students upload                                                                                       | deliberate about what the next course of action should be [coming soon]      |
-| `* *`    | eager to learn student                 | receive feedback (read comments and suggestions) from my professor                                                                      | set a direction to make progress for my FYP [coming soon]                    |
-| `* *`    | responsible student                    | update my FYP progress through the app                                                                                                  | inform my professor on my progress [coming soon]                             |
-| `* *`    | critical student                       | provide feedbacks on my FYP guidance and evaluation                                                                                     | help the FYP management team to further improve [coming soon]                |
+| `* *`    | eager student                          | receive feedback promptly, i.e. read comments and suggestions from my professor ASAP                                                    | set a clear direction for my FYP in advance [coming soon]                    |
+| `* *`    | responsible FYP student                | update my FYP progress through the app                                                                                                  | inform my professor on my progress [coming soon]                             |
+| `* *`    | critical student                       | provide feedbacks on my FYP guidance and evaluations by my professor                                                                    | help the FYP management team to further improve [coming soon]                |
 | `* *`    | helpful SoC professor                  | put tags on my student's FYP project so that I can let him/her know whether they are “Ahead of time”, “Up to task” or “Behind schedule” | inform my students of their progress relative to the timeline [coming soon]  |
```

Comment:
The benefit is duplicated inside the functionality

Reply:
I suppose you mean the duplicate "so that"? I'll fix that!